puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/hammerCliKatelloNightlyRpmBuild.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    option {
+        timestamps()
+        timeout(time: 30, unit: 'MINUTES')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+
+        stage ("Build Gem") {
+            steps {
+
+                dir('hammer_cli_katello') {
```

Comment:
It's typically better to have the first thing cloned not be in a directory unless you put all things cloned into individual directories.